run.sh

```diff
@@ -7,14 +7,43 @@ set -e
 mkdir /tmp/build
 cd /tmp/build
 
+BUILD_FILE="private-build-plans.toml"
+BUILD_PARAM="contents::iosevka"
+
+CUSTOM_BUILD_FILE=false
+
+# Check the input
+if [[ -f "/build/$BUILD_FILE" ]]; then
+    echo "Found custom build-plans file: $BUILD_FILE"
+    CUSTOM_BUILD_FILE=true
+    if [[ -z $1 ]]; then
+        # Get the name of the first build plan when the user does not provide
+        # custom build arguments (automatic mode)
+        PLAN_NAME=$(grep -Po -m 1 '(?<=buildPlans.)[^\]]*' /build/$BUILD_FILE)
+        BUILD_PARAM="contents::$PLAN_NAME"
+    else
+        # User knows what they are doing and provided custom build arguments
+        # (manual mode)
+        BUILD_PARAM="$1"
+    fi
+else
+    echo "Custom build-plans file not found, using the default one"
+    if [[ -n $1 ]]; then
+        echo -e "\nError: Custom build plan '$1' requested, but custom '$BUILD_FILE' file not found"
+        exit 1
```

Comment:
I'm thinking of a case when the above logic would prevent people from using the default build-plans file with custom build param. Currently the script would fail. Not sure if it's ok. Let's remove it.

Reply:
Yes, I guess you're right.. but what would be bad about leaving the error message?